### PR TITLE
[POC PYTHON API] Implement read_network 

### PR DESCRIPTION
**Pull request by AlexeyLebedev1**
### Tickets:
 - *51219*

### CODE REVIEW

**Comment by AlexeyLebedev1** [comment]
@jiwaszki Can you please review this PR? It seems I have no permissions in this repo to add you to the reviewers.

**Review by jiwaszki — CHANGES_REQUESTED** [review]

**Review by jiwaszki — APPROVED** [review]
Looks good!

**Review by jiwaszki — CHANGES_REQUESTED** [review]
Only one small code style change

**Review by jiwaszki — APPROVED** [review]

**Review by akuporos — CHANGES_REQUESTED** [review]
Please, add the next test which checks read network from buffer:
`    ie = IECore()
    with open(test_net_bin, 'rb') as f:
        bin = f.read()
    with open(model_path()[0], 'rb') as f:
        xml = f.read()
    net = ie.read_network(model=xml, weights=bin)
    assert isinstance(net, IENetwork)`

Looks like it'll fail.

File: ngraph/python/src/openvino/inference_engine/ie_api.py [hunk 1]
```diff
@@ -84,3 +84,10 @@ def __new__(cls, tensor_desc : TensorDesc, arr : np.ndarray = None):
             return TBlobUint8(tensor_desc, arr, arr_size)
         else:
             raise AttributeError(f'Unsupported precision {precision} for Blob')
+
+def blob_from_file(path_to_bin_file : str) -> BlobWrapper:
+    array = np.fromfile(path_to_bin_file,dtype=np.uint8)
```
Comment (akuporos) [inline ngraph/python/src/openvino/inference_engine/ie_api.py:89]: I'm not sure that weight's precision is always equal to "U8". Maybe one more parameter for this function should be added like `precision` with set default value to "U8" and, please, use the precision_map for getting numpy's dtype (the precision_map you can find above)

**Review by akuporos — CHANGES_REQUESTED** [review]

File: ngraph/python/src/openvino/inference_engine/ie_api.py (hunk 1 above)
Comment (jiwaszki) [reply ngraph/python/src/openvino/inference_engine/ie_api.py:89]: You should look at _openvino/inference-engine/src/inference_engine/ie_network_reader.cpp_ around the **line 217**.
`Blob::Ptr weights = make_shared_blob<uint8_t>({Precision::U8, { fileSize }, C });`
Looks like it is forcing it to U8 by default.

Of course we can use `dtype` attribute in `blob_from_file` and set it to default but I do not see a code responsible for other possible precisions inside IE.

Comment (akuporos) [reply ngraph/python/src/openvino/inference_engine/ie_api.py:89]: Ok, I discussed this with relevant people - a blob over weights can be created with U8 precision.

**Review by akuporos — APPROVED** [review]

**Pull request merged**